ocs_ci/ocs/registry.py

```diff
@@ -68,10 +68,10 @@ def get_registry_pod_obj():
             break
         elif len(pod_obj) == 0:
             raise UnexpectedBehaviour("Image-registry pod not present")
-        elif iteration > 5:
+        elif iteration == 18:  # 18*10 = 180 secs.
             raise UnexpectedBehaviour("Waited for 3 mins Image-registry pod is not in Running state")
         else:
-            logger.info(f"Waiting for 30 sec's for registry pod to be up iteration {iteration}")
+            logger.info(f"Waiting for 10 sec's for registry pod to be up iteration {iteration}")
```

Comment:
```suggestion
            logger.info(f"Waiting for {wait_time} sec's for registry pod to be up iteration {iteration}")
```

Reply:
Sure, I can change that. I was just going what whatever was there before.